app.js

```diff
@@ -6,3 +6,80 @@
   // get user input
   // use user input to update state 
   // update DOM to reflect the new state
+const userNameInput = document.getElementById('userName');
+const userPronounInput = document.getElementById('userPronoun');
+const submit = document.getElementById('submit');
+const userName = document.getElementById('name');
+const userPronouns = document.getElementById('pronouns');
+
+const buttonOne = document.getElementById('red');
+const buttonTwo = document.getElementById('blue');
+const buttonThree = document.getElementById('green');
+const buttonFour = document.getElementById('yellow');
+const tagColorOne = document.getElementById('greeting');
+const tagColorTwo = document.getElementById('bottom');
+
+submit.addEventListener('click', () => {
+    const value = userNameInput.value;
+    userName.textContent = value;
+});
+submit.addEventListener('click', () =>{
```

Comment:
You do not need multiple event listeners for the same button; you could consolidate this into one event listener.